### ISSUE TITLE
UX: Need a user to be able to experience the data dex without connecting a wallet

### ISSUE THREAD

**Issue by newbreedofgeek** [Itheum/data-dex#116]
Currently, the user HAS to connect a wallet to experience the data dex. This is not ideal. We need to let the user experience certain features without connecting a wallet and then expand the features once a wallet is connected.

A good candidate is the Data NFT marketplace (https://github.com/Itheum/data-dex/issues/25), once this is done we can work on this task.

**Comment by newbreedofgeek** [comment]
This will be part of the UI Refresh for V1 release

**Comment by newbreedofgeek** [comment]
Closing and will replace with new ticket
